### PR TITLE
Added accordion button with a coroutine animation

### PR DESCRIPTION
**Pull request by TinelValentin**

### CODE REVIEW

File: SevenSlots/SevenSlots/SevenSlots/Commands/AccordionAnimationCommand.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            return true;
+        }
+
+        /// <summary>
+        /// Calls a coroutine every 5 seconds which will move the button
+        /// </summary>
+        /// <param name="parameter"></param>
+        public void Execute(object parameter)
+        {
+            Device.StartTimer(TimeSpan.FromSeconds(5), () => Animationtimer(parameter as Button));
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/Commands/AccordionAnimationCommand.cs]: Animationtimer and not AnimationTimer?

File: SevenSlots/SevenSlots/SevenSlots/Commands/AccordionAnimationCommand.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        }
+
+        async Task MoveAwayAsync(Button button)
+        {
+            await button.TranslateTo(finalPoint, 0, animationTime);
+        }
+
+        bool Animationtimer(Button button)
+        {
+
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/Commands/AccordionAnimationCommand.cs]: Extra space?

File: SevenSlots/SevenSlots/SevenSlots/Commands/AccordionAnimationCommand.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            Device.BeginInvokeOnMainThread(async () =>
+            {
+
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/Commands/AccordionAnimationCommand.cs]: Extra space x2

File: SevenSlots/SevenSlots/SevenSlots/ViewModel/HomePageViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        }
+
+        public HomePageViewModel()
+        {
+            AccordionText = "TestValue";
+            AccordionAnimationCommand = new AccordionAnimationCommand();
+        }
+
+        
+
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/ViewModel/HomePageViewModel.cs]: A lot of extra space

**Review by Sorin37 — CHANGES_REQUESTED** [review]

**Review by BrassoiCatalin — APPROVED** [review]

**Review by Sorin37 — APPROVED** [review]

**Pull request merged**